src/Lib/Configuration.php

```diff
@@ -455,6 +461,37 @@ public function setJsonOptions(int $jsonOptions)
         return $this;
     }
 
+    /**
+     * @return string
+     */
+    public function getConnectionName(): string
+    {
+        return $this->connectionName;
+    }
+
+    /**
+     * @param string $connectionName Connection name to use when loading tables for building schemas from models.
+     * @return $this
+     */
+    public function setConnectionName(string $connectionName)
+    {
+        $configuredConnections = ConnectionManager::configured();
```

Comment:
Is this check too heavy? Should we just check if it's a valid string instead? Wasn't sure.

Reply:
No, this is fine.